src/config.ts

```diff
@@ -217,37 +254,95 @@ const defaultConfig: IConfig = {
     enabled: false,
   },
   experimentalRustCrypto: false,
-
-  // Needed to make the interface happy.
-  RUNTIME: {},
+  configMeta: undefined,
 };
 
 export function getDefaultConfig(): IConfig {
   return Config.util.cloneDeep(defaultConfig);
 }
 
+function logNonDefaultConfiguration(config: IConfig): void {
+  log.info(
+    "non-default configuration properties:",
+    JSON.stringify(getNonDefaultConfigProperties(config), null, 2)
+  );
+}
+
+function logConfigMeta(config: IConfig): void {
+  log.info("Configuration meta:", JSON.stringify(config.configMeta, null, 2));
+}
```

Comment:
@MTRNord @FSG-Cat your review has been requested because I just wanted you to see and give thoughts on the log output shown in the PR description

Reply:
Meow when is this shown?

Regular exit via Systemd shutting the bot down or only times when the bot shuts down unexpectedly.

Reply:
We don't intercept either SIGINT or SIGTERM cos that sounded kinda sketchy. It's only when draupnir crashes does the stuff get logged.

Reply:
This behavior of only showing on crash makes sense to me.